metis-debias/metis-debias-detect-service/src/main/java/eu/europeana/metis/debias/detect/model/request/DetectionParameter.java

```diff
@@ -0,0 +1,51 @@
+package eu.europeana.metis.debias.detect.model.request;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * The type Detection parameter.
+ */
+public class DetectionParameter {
```

Comment:
Wondering if there is perhaps a more descriptive name? I can imagine something like `DetectionInput` or `LiteralValuesForDetection` or something like that?

Reply:
I used `BiasInputLiterals`

Reply:
OK!